### PR TITLE
Create Auto_Restart_Launcher.bat

### PR DESCRIPTION
**Pull request by CoreyUK**
Automated bat that will relaunch server when crashed

### CODE REVIEW

File: Auto_Restart_Launcher.bat [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+set name=BOIII Nameofserver
+
+:://////////////////////////////////////////////////////////////////////////
+:://      The regex search for the window name of the sever. (optional)   //
+:://                                                                      //
+:://     This will help identify which window needs closing in case the   //
+:://     server errors out or freezes according to filename on g_log      //
+:://                                                                      //
+:://     If hosting multiable. I would suggest putting your hostname      //
```
Comment (Dss0) [inline Auto_Restart_Launcher.bat]: shouldn't it be "multiple"?
"If hosting multiple servers i suggest putting your hostname.."

File: Auto_Restart_Launcher.bat [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+:://          Example: Default BOIII Server  on mp_                       //
```
Comment (Dss0) [inline Auto_Restart_Launcher.bat]: on mp_? Also one too many spaces

File: Auto_Restart_Launcher.bat [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+:://////////////////////////////////////////////////////////////////////////
+:://  DONE!! WARNING! Don't mess with anything below this line. SEROUSLY! //
+:://             Only edit if you know what you doing!!!                  //
+:://////////////////////////////////////////////////////////////////////////
+
+// Create a copy of boiii.exe and rename it to something like boiiitdm.exe then edit the .exe names below to suit!
+title %name% - Auto Restart
+echo (%date%)  -  (%time%) Starting %name% with port %GamePort% with %ServerFilename%....
+start boiiitdm.exe -dedicated +set fs_game "%ModFolderName%" +set net_port "%GamePort%" +set logfile 2 +exec %ServerFilename%   //this line
```
Comment (Dss0) [inline Auto_Restart_Launcher.bat:88]: what's boiiitdm.exe? shouldn't it be boiii.exe?

File: Auto_Restart_Launcher.bat [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+	
+	dir "%log_path%"\"%log_file%" > nul
+	for /f "delims=" %%i in ('"forfiles /p "%log_path%" /m "%log_file%" /c "cmd /c echo @ftime" "') do set modif_time_temp=%%i
+
+	if "%modif_time_temp%" == "%modif_time%" (
+		echo "(%date%)  -  (%time%) WARNING: %name% hung, killing server..."            
+		::https://stackoverflow.com/questions/26552368/windows-batch-file-taskkill-if-window-title-contains-text
+		for /f "tokens=2 delims=," %%a in ('
+			tasklist /fi "imagename eq boiiitdm.exe" /v /fo:csv /nh                                  //this line
+			^| findstr /r /c:"boiiitdm.exe"                                                            //this line
```
Comment (Dss0) [inline Auto_Restart_Launcher.bat:109]: same as above

**Review by Dss0 — COMMENTED** [review]
tbh i don't like this pr, it's difficult to configure for unexperienced users and doesn't really belong into this repo.

File: Auto_Restart_Launcher.bat (hunk 3 above)
Comment (CoreyUK) [reply Auto_Restart_Launcher.bat:88]: Reason for the renamed exe per server, Is due to the current crashing situation that is on going, if one server goes down with the same .exe name it tanked the rest of my servers

**Pull request closed without merging**